nvssclient/Services/TimedHostedService.cs

```diff
@@ -134,13 +134,19 @@ public async void SubmitNewMessages()
                         context.Update(item);
                         context.SaveChanges();
                     }
-                    else if (response.StatusCode == HttpStatusCode.Unauthorized)
-                    {
-                        _logger.LogError($">>> Unauthorized error submitting {message.MessageId}, status: {response.StatusCode}");
-                    }
                     else
                     {
-                        _logger.LogError($">>> Error submitting {message.MessageId}, status: {response.StatusCode}");
+                        if (response.StatusCode == HttpStatusCode.Unauthorized)
+                        {
+                            _logger.LogError($">>> Unauthorized error submitting {message.MessageId}, status: {response.StatusCode}");
+                        }
+                        else
```

Comment:
We only want to set the status code to error if there was something wrong with the submitted data and the HttpStatusCode was a 400, see here for different error responses https://github.com/nightingaleproject/Reference-NCHS-API#http-error-responses

If the user was unauthorized (maybe needs a token refresh), or the server was temporarily down and returned 404, we want the data to get resubmitted